pkg/config/config.go

```diff
@@ -294,10 +299,19 @@ func (cd *configuration) load(cm *corev1.ConfigMap) {
 		cd.enableDefaultRegistryMutation = newEnableDefaultRegistryMutation
 	}
 	// load excludeGroupRole
-	cd.excludeGroupRole = append(cd.excludeGroupRole, parseRbac(cm.Data["excludeGroupRole"])...)
-	cd.excludeGroupRole = append(cd.excludeGroupRole, defaultExcludeGroupRole...)
+	excludedGroups, ok := cm.Data["excludeGroupRole"]
```

Comment:
Do we want to rename this configmap var to `excludedGroups` to make it consistent?

Reply:
Ideally yes but it's a breaking change.

Reply:
Let's update the [doc](https://kyverno.io/docs/installation/#configmap-flags) first, currently the defaults are 
```
Default we are using system:serviceaccounts:kube-system,system:nodes,system:kube-scheduler.
```

`system:kube-scheduler` is moved to `excludedUsernames`.

Reply:
@eddycharly - do you agree ⬆️? I can log an issue for this.